### PR TITLE
♻️ Updated Switch tokens

### LINKED ISSUE #1258: Conform Switch tokens

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1258

### CODE REVIEW

File: libraries/core-react/src/components/SelectionControls/Switch/Input.tsx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import { forwardRef, InputHTMLAttributes } from 'react'
 import styled from 'styled-components'
-import { switchControl as tokens } from './Switch.tokens'
+import { comfortable as tokens } from './Switch.tokens'
 import type { Size } from './Switch.types'
+import { outlineTemplate } from '../../../utils'
 
-const { enabled, disabled: _disabled } = tokens
+// const { enabled, disabled: _disabled } = tokens
```
Comment (pomfrida) [inline libraries/core-react/src/components/SelectionControls/Switch/Input.tsx:7]: Remove comment

File: libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts [hunk 2]
```diff
@@ -10,69 +11,114 @@ const {
...
-    clickSize: clickbounds,
+
+type SwitchToken = ComponentToken
+
+export const comfortable: SwitchToken = {
+  typography: labelTypography,
+  clickbound: {
+    height: clickbounds.default__base,
+    width: clickbounds.default__base,
+    offset: { top: '0' },
```
Comment (pomfrida) [inline libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts:36]: I saw you made offset optional so I think this isn't needed anymore?

File: libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts [hunk 3]
```diff
@@ -10,69 +11,114 @@ const {
...
+        handle: {
           background: primaryHover,
+          // small: {
+          //   background: primaryHoverAlt,
+          // },
+        },
+        track: {
+          // small: {
+          //   background: primaryHover,
+          // },
```
Comment (pomfrida) [inline libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts:119]: Remove comments here?

**Review by pomfrida — CHANGES_REQUESTED** [review]
Just a few small things to remove

File: libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts [hunk 4]
```diff
@@ -10,69 +10,106 @@ const {
...
-      track: {
-        small: {
+    },
+    hover: {
+      background: primaryHoverAlt,
+      entities: {
+        handle: {
           background: primaryHover,
         },
+        track: {},
```
Comment (pomfrida) [inline libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts:111]: Do we really need this now though? 🤔

Comment (mimarz) [reply libraries/core-react/src/components/SelectionControls/Switch/Switch.tokens.ts:111]: If its not used remove it! 🔥

**Review by pomfrida — APPROVED** [review]
LGTM 👍

**Pull request merged**